apps/web/src/pages/ProfileAndApplicationsPage/ProfileAndApplicationsPage.tsx

```diff
@@ -24,10 +30,12 @@ export const ProfileAndApplications = ({
   const intl = useIntl();
   const applications = user.poolCandidates?.filter(notEmpty) ?? [];
 
+  const formattedDescription = intl.formatMessage(description);
   return (
     <>
       <SEO
         title={intl.formatMessage(navigationMessages.profileAndApplications)}
+        description={formattedDescription}
```

Comment:
Why do we need the vars `description` and formattedDescription, can't we just do this?
```suggestion
        description={intl.formatMessage({
          defaultMessage:
            "Manage your personal information, career timeline, skills, and track applications.",
          id: "OyV6MH",
          description: "SEO description for profile and applications hero",
        })}
```

Reply:
Ya for this one we don't need to create a constant since it's only being used once. Fixed by [b662094](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9839/commits/b66209496a910347286c9d4dd2245f267d4614af)

Reply:
Half way there, I think. See: https://github.com/GCTC-NTGC/gc-digital-talent/pull/9839#discussion_r1535574785